crates/types/src/services/p2p.rs

```diff
@@ -44,18 +44,15 @@ pub struct GossipsubMessageInfo {
     pub peer_id: PeerId,
 }
 
-// TODO: Maybe we can remove most of types from here directly into P2P
```

Comment:
Removed this comment because it's not clear exactly what it refers to, and we don't have an issue to track it.